<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1">
		<title>William Owens (Biologist, Programmer, Engineer): Research</title>
		<link href="main.css" rel="stylesheet">
    </head>
    <body>
        <div class="wrapper">
            <div class="wrapper-inner">
                <nav>
                    <div class="nav-inner">
                        <div class="nav-button">
                            <a href="/">About</a>
                        </div>
                        <div class="nav-button">
                            <a href="projects.html">Projects</a>
                        </div>
                        <div class="nav-button">
                            <a class="current">Research</a>
                        </div>
                        <div class="nav-button">
                            <a href="etcetera.html">Et cetera</a>
                        </div>
                    </div>
                    <div class="home-button">
                        <a href="/" style="color: inherit;">William Owens</a>
                    </div>
                </nav>
            </div>
            <div class="wrapper-inner">
                <h1>Research</h1>
                <hr>
                <div> 
                    <h2 class="align-left">Kladde Lab</h2>
                    <h2 class="align-right">(Undergraduate Researcher, 2018-2020)</h2>
                </div>
                <div style="clear: both;"></div>
                <p>
                    I first contacted <a href=https://biochem.med.ufl.edu/research/primary-faculty/michael-kladde/>Dr. Michael Kladde</a> in fall 2017 and began serious work in his lab the following spring.
                    The Kladde Lab explores the relation between epigenetics and cancer.
                    Our work revolves around MAPit, an epigenetic assay that profiles both chromatin accessiblity and DNA methylation patterns simultaneously.
                    By using innovative techniques, the Kladde Lab traces the sequence of epigenetic events that lead to oncogenesis (cancer development).
                    For more information about the Kladde Lab's work, you can consult the <a href=https://kladde.biochem.med.ufl.edu/research-description/>lab's website</a>.
                </p>
                <br>
                <p>
                    My experience as a computer science student proved indispensible in the lab.
                    Because the Kladde Lab relies upon novel experimental strategies and produces vast quantities of data, we needed sophisticated, powerful computational techniques to analyze our data.
                    For this demanding task, the lab uses <a href="https://www.rc.ufl.edu/services/hipergator/">HiPerGator 2.0</a>, the University of Florida supercomputer.
                    For my data analysis duties, I processed critical data for all other members of the Kladde Lab.
                    I have wrote dozens of scripts, contributed to open source bioinformatic pipelines, and wrote new bioinformatic programs of my own.
                </p>
                <br>
                <p>
                    Beyond my work <i>in silico</i>, I moved to wet lab work for my Honors thesis, <i>Investigation of a Putative, Distal Mismatch Repair Gene Enhancer</i>.
                    For this project, I looked at a region that posessed several epigenetic markers suggestive of an enhancer.
                    Based on known single nucleotide polymorphisms (SNPs) in this region, I hypothesized that this region was an enhancer for an important mismatch repair gene and mutations in this region could serve as a biomarker for certain cancers.
                    I tested this hypothesis using CRISPR/cas9, deleting the region and observing the effects on the MMR gene's transcription.
                    (My thesis is embargoed in anticipation of a future publication. If you have questions about this project, feel free to email or DM me.)
                </p>
                <hr>
                <div> 
                    <h2 class="align-left">iGEM Team UFlorida 2018</h2>
                    <h2 class="align-right">(Founding Member, 2017-2018)</h2>
                </div>
                <div style="clear: both;"></div>
                <div class="normal-inset">
                    <img src="images/igem-team.jpg" alt="Team UFlorida 2018" style="width: 100%">
                </div>
                <br>
                <p>
                    The International Genetically Engineered Machines competition (iGEM) is held yearly to populate BioBrick repositories with useful biological parts.
                    A biological part is simply a DNA sequence that encodes a product with a biological function.
                    In this respect, iGEM is part of the growing movement that applies the principles of engineering to the field of biology.
                </p>
                <br>
                <p>
                    With a strong pre-health background, Team UFlorida was primarily interested in therapeutic applications.
                    After a series of false starts and difficulties finding proper lab space, we decided to engineer a gut bacterium to produce copious amounts of butyrate, a short chain fatty acid (SFCA) with known probiotic benefits.
                    For our bacterium, we selected <i>Escherichia coli</i> strain <i>Nissle 1917</i>, a probiotic bacterium that can colonize the human gut.
                    For more information on our project, you can refer to our <a href="http://2018.igem.org/Team:UFlorida">iGEM wiki</a>.
                </p>
                <br>
                <p>
                    In short, we deleted two competiting genes from <i>Nissle 1917</i> and better optimized it for butyrate synthesis.
                    We innovated on a previous design submitted to the iGEM repository and successfully reported marginal butyrate production, a first in iGEM history.
                    For our work, Team UFlorida received a Silver Medal, the first medal for our university.
                    Other teams have expressed interest in our <i>Nissle 1917</i> mutant, and we hope they can take our project even further.
                </p>
                <br>
                <p>
                    As for my specific contributions, I applied my diverse skillset to several different tasks.
                    I played a large part during the design phase and wrote a few scripts to aid in designing codon-optimized operons for <i>Nissle 1917</i>.
                    I also worked at the bench as we attempted to transform <i>Nissle 1917</i> with our Butyrate synthesis pathway and helped revise our strategy when we encountered obstacles.
                    For more details on designing the pathway, you can refer to <a href="documents/wso-usp-poster.pdf">this poster</a> that I presented at the 2019 Undergraduate Research Symposium at UF.
                </p>
                <br>
                <p>
                    Calling this project "chaotic" would be an understatement.
                    Nevertheless, this project was a major highlight of my undergraduate career.
                    From this work, I was exposed to the parts of research that most undergraduates do not see and gained invaluable experience doing real independent work.

                </p>
                <br>
                <hr>
                <div> 
                    <h2 class="align-left">Bass Lab</h2>
                    <h2 class="align-right">(Undergraduate Research Assistant, 2017)</h2>
                </div>
                <div style="clear: both;"></div>
                <br>
                <p>
                    I spent my summer after high school graduation volunteering in <a href="https://www.bio.fsu.edu/faculty.php?faculty-id=bass">Dr. Hank Bass's</a> lab at Florida State University.
                    The following summer (2017), I returned as an undergraduate research assistant.
                    For the latest information on the Bass Lab, you can refer to <a href="https://www.bio.fsu.edu/~bass/">the lab's website</a>.
                </p>
                <br>
                <p>
                    During my second summer with the Bass Lab, I divided my time between two starkly different areas.
                    First, I worked in the FSU Research Fields to prepare the <a href="https://www.bio.fsu.edu/bass/maizenucleosome/outreach/">Maize-10-Maze</a>, an outreach event that allowed the public to walk through a recreation of the Maize genome.
                </p>
                <br>
                <p>
                    Second, I worked with Dr. Bass and his graduate student to test and implement a program for finding G4 quadruplexes <i>in silico</i>.
                    Here, I applied my knowledge as a budding programmer for the first time.
                    During my two summers in the Bass lab, I realized that this field desperately needed more biologists who code.
                    For this reason, I first signed up for programming classes at UF, and later declared computer science as my second major.
                </p>
                <br>
                <p>
                    Computer science has infected my view of the world; I don't want to be a biologist who codes but rather a <i>computational biologist</i>.
                    Years later, I can credit the Bass Lab for my decision to plunge into computer science, the best decision I have ever made.
                </p>
                <br>
                <hr>
            </div>
        </div>
    </body>
</html>
